internal/liqonet/network-manager/netcfgcreator/networkconfig.go

```diff
@@ -229,6 +229,6 @@ func (ncc *NetworkConfigCreator) EnforceNetworkConfigAbsence(ctx context.Context
 		return err
 	}
 
-	klog.Errorf("Correctly ensured no NetworkConfigs associated with cluster %s are present", clusterIdentity)
+	klog.Infof("Correctly ensured no NetworkConfigs associated with cluster %s are present", clusterIdentity)
```

Comment:
How often is it printed? Should we have a verbosity level set?

Reply:
This is printed only in case an object was present and it has been requested to be deleted. For this reason, if you agree, I would keep the current verbosity level.